docs/whats_new.rst

```diff
@@ -41,6 +41,8 @@ Bug
 
 API
 ~~~
+- Added two attributes to ``PrepPipeline``: ``noisy_channels_before_interpolation`` and ``noisy_channels_after_interpolation`` which have the detailed output of each noisy criteria, by `Yorguin Mantilla`_ (`#45 <https://github.com/sappelhoff/pyprep/pull/45>`_)
+- Added two keys to the ``PrepPipeline.noisy_channels_original`` dictionary: ``bad_by_dropout`` and ``bad_by_SNR``, by `Yorguin Mantilla`_ (`#45 <https://github.com/sappelhoff/pyprep/pull/45>`_)
```

Comment:
Please put this in `Changelog`. In `API` we should list only **changes** to the existing API, so that users can check that section and see where they MUST adjust their code in order to continue working.

Your contribution in this PR is more like a new feature, which we list in `Changelog`.

Lastly, you can refer to `PrepPipeline` using the RST syntax:

```
:class:`PrepPipeline`
```

That will make it possible to link from the Changelog section of the docs directly to the API section ... it just doesn't work right now (don't know why) --- but that's a different fix.

See the difference here: https://pyprep--45.org.readthedocs.build/en/45/whats_new.html

![image](https://user-images.githubusercontent.com/9084751/101238034-af339e00-36dd-11eb-840a-2c18deccedd8.png)

those methods and classes that have been documented with a leading `:meth:` or `:class:` respectively are printed bold ... and if everything were working, they would link to the API section. You can also do it for functions using `:func:`. 🙂 the beauty of sphinx+rst

I opened https://github.com/sappelhoff/pyprep/issues/46 for this.